ports/stm32/mboot/README.md

```diff
@@ -89,6 +89,16 @@ How to use
    the beginning of the chunk when the end is reached.  Then use a split
    raw filesystem to inform mboot of this wrapping.
 
+   The version and config options that mboot was built with are stored in at 
+   the end of the mboot flash block. The length of this is section defaults
```

Comment:
"The version and config options that mboot was built with are stored in a small, fixed section of bytes at the end of the flash region allocated for mboot.  The length of the fixed section defaults ..."